src/helpers/helper.js

```diff
@@ -43,3 +48,7 @@ export const getFavouritesList = (contactsList) => {
   }
   return result;
 };
+
+const areContactsEqual = (c1, c2) => c1.firstName + c1.lastName === c2.firstName + c2.lastName || c1.image === c2.image;
```

Comment:
Whenever you want to concatenate strings, use the ES6 string format (also called Template Strings):

``` `${c1.firstName} {c2.lastName}` ```
this is the same as having c1.firstName + ' ' + c1.lastName